components/permission-controller/controllers/namespace_controller_test.go

```diff
@@ -0,0 +1,221 @@
+package controllers_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	"github.com/kyma-project/kyma/components/permission-controller/controllers"
+)
+
+const (
+	timeout         = 5 * time.Second
+	systemNamespace = "test-system"
+	adminGroup      = "namespace-admins"
+)
+
+var _ = Describe("Permissions Controller", func() {
+	Context("should watch for namespace creation and", func() {
+		Context("create rolebinding if the namespace is not a system one and", func() {
+			It("add static user to subjects if UseStaticConnector is set to true", func() {
+
+				testRolebindingName, testNamespaceName := controllers.RolebindingName, "dev"
+				expectedRequest := &reconcile.Request{NamespacedName: types.NamespacedName{Name: testNamespaceName}}
+
+				s := scheme.Scheme
+				err := rbac.AddToScheme(s)
+				Expect(err).NotTo(HaveOccurred())
+
+				err = corev1.AddToScheme(scheme.Scheme)
```

Comment:
I think this should be: 
```suggestion
				err = corev1.AddToScheme(s)
```